### PR TITLE
new configuration options

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: aiven/templates/service_user_config_schema.json [hunk 1]
```diff
@@ -1,22 +1,237 @@
 {
+  "alerta": {
```
Comment (rikonen) [inline aiven/templates/service_user_config_schema.json]: This is internal and must not be included here.

File: aiven/templates/service_user_config_schema.json [hunk 2]
```diff
@@ -1256,28 +1700,501 @@
     },
     "type": "object"
   },
+  "kafka_mirrormaker": {
```
Comment (rikonen) [inline aiven/templates/service_user_config_schema.json]: I would not include this or the m3 related settings as those may still change before making them generally available.

File: aiven/templates/service_user_config_schema.json [hunk 3]
```diff
@@ -1959,5 +3034,174 @@
       }
     },
     "type": "object"
+  },
+  "sw": {
```
Comment (rikonen) [inline aiven/templates/service_user_config_schema.json]: Internal, must not be included

File: aiven/templates/service_user_config_schema.json [hunk 4]
```diff
@@ -1959,5 +3034,174 @@
...
+          "type": "object"
+        },
+        "default": {},
+        "title": "Aiven Web service settings",
+        "type": "object"
+      }
+    },
+    "type": "object"
+  },
+  "zookeeper": {
```
Comment (rikonen) [inline aiven/templates/service_user_config_schema.json]: Will be removed soon, must not be included

**Comment by ivan-savciuc** [comment]
I am running acceptance tests with the new configuration options ...

**Pull request merged**